client/components/status/status.js

```diff
@@ -722,7 +722,11 @@ export default class Status extends React.Component {
               ref={this.loginFormRef}
               method={captivePortalLoginForm.method || "post"}
               id="cp-login-form"
-              action={captivePortalLoginForm.action || ""}
+              action={
+                process.env.NODE_ENV !== "development"
+                  ? captivePortalLoginForm.action || ""
+                  : "http://localhost:8000/captive-portal-mock/login/"
```

Comment:
This would mean that if I want to test a real captive portal in a development environment I won't be able to do it.

I suggest just adding these as default values in the default config:

- https://github.com/openwisp/openwisp-wifi-login-pages/blob/master/org-configurations/default-configuration.yml#L421-L442
- https://github.com/openwisp/openwisp-wifi-login-pages/blob/master/internals/generators/organization/index.js#L59-L81